### PR TITLE
Add missing options to Basic Options

### PR DESCRIPTION
**Pull request by sentrivana**
## Pre-merge checklist

*If you work at Sentry, you're able to merge your own PR without review, but please don't unless there's a good reason.*

- [x] Checked Vercel preview for correctness, including links
- [x] PR was reviewed and approved by any necessary SMEs
- [ ] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

## Description of changes

`include_source_context` and `include_local_variables` were missing from the Basic Options page.

## Preview

- https://sentry-docs-f5f6onk55.sentry.dev/platforms/python/configuration/options/#include-source-context

### CODE REVIEW

**Review by antonpirker — APPROVED** [review]

File: src/platforms/common/configuration/options.mdx [hunk 1]
```diff
@@ -248,6 +248,22 @@ If <PlatformIdentifier name="send-default-pii" /> is `off`, scrubs the event pay
...
+
+You can opt out of source context being included in events sent to Sentry by disabling this option.
+
+The option is enabled by default.
+
+</ConfigKey>
+
+<ConfigKey name="include-local-variables" supported={["python"]}>
+
+If this option is disabled, the SDK will not capture a snapshot of local variables to send with the event.
```
Comment (antonpirker) [inline src/platforms/common/configuration/options.mdx:261]:
```suggestion 
If this option is enabled, the SDK will capture a snapshot of local variables to send with the event.
```

File: src/platforms/common/configuration/options.mdx [hunk 2]
```diff
@@ -248,6 +248,22 @@ If <PlatformIdentifier name="send-default-pii" /> is `off`, scrubs the event pay
 
 </ConfigKey>
 
+<ConfigKey name="include-source-context" supported={["python"]}>
+
+You can opt out of source context being included in events sent to Sentry by disabling this option.
```
Comment (antonpirker) [inline src/platforms/common/configuration/options.mdx:253]:
```suggestion
If the source context is being included in events sent to Sentry. The source context are the five lines of code above and below the line of code where an error was happening. 
```

File: src/platforms/common/configuration/options.mdx [hunk 3]
```diff
@@ -248,6 +248,22 @@ If <PlatformIdentifier name="send-default-pii" /> is `off`, scrubs the event pay
 
 </ConfigKey>
 
+<ConfigKey name="include-source-context" supported={["python"]}>
+
+If this option is enabled, the source context will be included in events sent to Sentry. The source context are the five lines of code above and below the line of code where an error was happening.
```
Comment (shanamatthews) [inline src/platforms/common/configuration/options.mdx:253]:
```suggestion
When enabled, source context will be included in events sent to Sentry. This source context includes the five lines of code above and below the line of code where an error happened.
```

File: src/platforms/common/configuration/options.mdx [hunk 4]
```diff
@@ -248,6 +248,22 @@ If <PlatformIdentifier name="send-default-pii" /> is `off`, scrubs the event pay
...
+
+The option is enabled by default.
+
+</ConfigKey>
+
+<ConfigKey name="include-local-variables" supported={["python"]}>
+
+If this option is enabled, the SDK will capture a snapshot of local variables to send with the event to help with debugging.
```
Comment (shanamatthews) [inline src/platforms/common/configuration/options.mdx:261]:
```suggestion
When enabled, the SDK will capture a snapshot of local variables to send with the event to help with debugging.
```

**Review by shanamatthews — APPROVED** [review]
Just a couple of tweaks to make the page's grammar more consistent.

**Pull request merged**